odahuFlow/operator/pkg/odahuflow/k8s.go

```diff
@@ -33,6 +33,7 @@ const (
 	PackagerResultStep        = "result"
 	TrainerSetupStep          = "setup"
 	TrainerTrainStep          = "trainer"
+	TrainerValidationStep     = "validation"
```

Comment:
Please, add the step into this array https://github.com/odahu/odahu-flow/blob/develop/odahuFlow/operator/pkg/repository/packaging/kubernetes/model_packaging.go#L47